src/wallet/scriptpubkeyman.cpp

```diff
@@ -2291,7 +2291,7 @@ bool DescriptorScriptPubKeyMan::SetupDescriptorGeneration(const CExtKey& master_
 
     // Mainnet derives at 0', testnet and regtest derive at 1'
     if (Params().IsTestChain()) {
-        desc_prefix += "/1'";
+        desc_prefix += "/1h";
     } else {
         desc_prefix += "/0'";
```

Comment:
Missed a `'` here and one below at line 2300.

Reply:
Thanks, I also updated the `combo()` descriptor in `MigrateToDescriptor`.